kitsune/settings.py

```diff
@@ -1021,3 +1021,40 @@ def read_only_mode(env):
         'environment': config('SENTRY_ENVIRONMENT', default=''),
     }
 }
+
+# Dead Man Snitches
```

Comment:
That is a whole lotta snitches... We need to find another way to monitor things like this. We just have too many.